### PR TITLE
Add async function test function.

### PR DESCRIPTION
**Pull request by solarhell**

### CODE REVIEW

**Comment by solarhell** [comment]
Maybe need some tweak for CI because only node8 supports native async function.

File: test/is.test.js [hunk 1]
```diff
@@ -90,6 +90,25 @@ describe('is', () => {
     });
   });
 
+  describe('asyncFunction', () => {
+    it('should true', () => {
+      var asyncFun = async function() {
```
Comment (dead-horse) [inline test/is.test.js]: test will failed in node < 7.6, so you should create a standalone test file, detect node version and require it.

Comment (solarhell) [inline test/is.test.js]: @dead-horse Now it passed all tests. Maybe you can help rebasing before merging it into master. Thanks.

**Comment by dead-horse** [comment]
1.2.0

**Pull request merged**